.github/workflows/pypi_release.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow will upload a Python Package to Pypi using Twine when a release is created in this Github repo.
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
+# Note: There is a bug in Github Actions, so do NOT use the “Save Draft” functionality when creating a new release: https://github.community/t/workflow-set-for-on-release-not-triggering-not-showing-up/16286/5
+# Remember to always verify tagged releases are actually available on the Pypi website: https://pypi.org/project/autogluon/
+
+name: Pypi Release
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3.3.0
```

Comment:
Can we just do `@v3` as it is highlighted in tutorials on the [action github](https://github.com/actions/checkout)?

Reply:
Well, if you click use latest button on this page: https://github.com/marketplace/actions/checkout you get what I put there. I think it doesn't matter to much anyways...